frontend/src/core/user/models/sessionModel.js

```diff
@@ -67,10 +67,12 @@ define(function(require) {
 
           Origin.trigger('login:failed', errorCode);
         }
+      }).always(function(){
+        self.set('loginFired',false);
```

Comment:
yes ,this is relevant to fix this issue as it helps to set flag on ajax request completion.